pkg/skaffold/runner/label/label.go

```diff
@@ -38,6 +41,26 @@ import (
 	patch "k8s.io/apimachinery/pkg/util/strategicpatch"
 )
 
+type withLabels struct {
+	deploy.Deployer
+
+	labels map[string]string
+}
+
+// WithLabels creates a deployer that sets labels on deployed resources.
+func WithLabels(d deploy.Deployer, labels map[string]string) deploy.Deployer {
```

Comment:
Seems a little strange to me that this lives in the `label` package, could we have `deploy.WithLabels(...)` instead?

Reply:
I used to have `with*` in deploy and build and @r2d4 made me move them to runner. lol.
Either way make me happy.

Reply:
lol. I put the `labels` package under `runner` because it's what was kicking off the actual labeling before, but if we're going to have the labeller embedded in the deployer itself I think we should move it out. @r2d4 WDYT